src/app/components/score-chart/score-chart.component.ts

```diff
@@ -1,19 +1,64 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input, OnChanges, OnInit, SimpleChanges } from '@angular/core';
 import * as Highcharts from 'highcharts';
+import more from 'highcharts/highcharts-more';
+import { parseISO } from 'date-fns';
+import { mean } from 'lodash';
+
+more(Highcharts);
 
 @Component({
   selector: 'app-score-chart',
   templateUrl: './score-chart.component.html',
   styleUrls: ['./score-chart.component.sass']
 })
-export class ScoreChartComponent implements OnInit {
+export class ScoreChartComponent implements OnChanges {
 
-  private options: any = {};
+  @Input()
+  public historicalData: string[];
```

Comment:
You should refine this type, as it is not clear what the data representation is like.
For example,

```typescript
interface Score {
    accentScore: number
    clarityScore: number
}

historicalData: Map<Date, Score>   // date as keys -> score as values
```

Reply:
Thats true. Will change!